server/market/market.go

```diff
@@ -41,6 +43,7 @@ const (
 	ErrInvalidCommitment      = Error("order commitment invalid")
 	ErrEpochMissed            = Error("order unexpectedly missed its intended epoch")
 	ErrDuplicateOrder         = Error("order already in epoch") // maybe remove since this is ill defined
+	ErrQuantityTooHigh        = Error("order quantity exceeds user limit")
```

Comment:
@dajohi An RPC error to look out for when submitting orders.  We can discuss when this is expected elsewhere.